pt-BR/mail/chrome/messenger/importDialog.dtd

```diff
@@ -67,13 +67,13 @@ LOCALIZATION NOTE : Do not translate any of the occurrences of the word
 
 <!ENTITY select.label             "Ou selecione o que deseja importar:">
 
-<!ENTITY title.label              "Title">
+<!ENTITY title.label              "Título">
 <!ENTITY processing.label         "Importando…">
 
 <!ENTITY FieldDiscInputText1.label  "O Netscape Communicator 4.x possui um endereço para cada contato, enquanto o &brandShortName; possui dois (residencial e comercial).">
 <!ENTITY FieldDiscInputText2.label  "Selecione a categoria na qual deseja amazenar os endereços importados:">
 <!ENTITY FieldDiscWarning.label  "Esta categoria será usada para todos os nomes do catálogo de endereços selecionado: ">
-<!ENTITY importDialog.homeTitle  "Home">
-<!ENTITY importDialog.workTitle  "Work">
+<!ENTITY importDialog.homeTitle  "Residencial">
+<!ENTITY importDialog.workTitle  "Comercial">
```

Comment:
Agreed. I did that, but I kept "Trabalho" for a few things, such as "Conta de Casa, Trabalho, etc.".